CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for your interest in contributing to Grafonnet! We welcome all people
+who want to contribute in a healthy and constructive manner within our
+community. Grafonnet is developed within the Grafana community. Therefore we
+are following the same [Code of Conduct as
+Grafana](https://github.com/grafana/grafana/blob/master/CODE_OF_CONDUCT.md). To
+help us create a safe and positive community experience for all, we require all
+participants to adhere to it.
+
+## Plugin Scope
+
+**Grafonnet only aims to support official plugins.**
```

Comment:
No, it only supports features of core Grafana. Even official plugins will need their own library.

Reply:
Changed to "core features and plugins"